includes/class-newspack-newsletters-subscription.php

```diff
@@ -301,5 +337,411 @@ public static function add_contact( $contact, $lists = [] ) {
 
 		return $result;
 	}
+
+	/**
+	 * Update a contact lists subscription.
+	 *
+	 * This method will remove the contact from all subscription lists and add
+	 * them to the specified lists.
+	 *
+	 * @param string   $email Contact email address.
+	 * @param string[] $lists Array of list IDs to subscribe the contact to.
+	 *
+	 * @return bool|WP_Error Whether the contact was updated or error.
+	 */
+	public static function update_contact_lists( $email, $lists = [] ) {
```

Comment:
Nit: this and a few other methods could be `private` to signal that they are internal, and make refactoring easier in the future.

Reply:
Good call, done in 1928278d3f0c9b9f30e3d809fd11159a96403b4d